src/PokemonGymOwner.java

```diff
@@ -0,0 +1,51 @@
+//import java.util.List;
+//
+//public class PokemonGymOwner {
+//
+//    String name;
+//    String town;
+//    List<Pokemon> pokemons;
+//
+////    public PokemonGymOwner(String name, String town, List<Pokemon> pokemons) {
+////    }
+//
+//
+//    public String getName() {
+//        return name;
+//    }
+//
+//    public String getTown() {
+//        return town;
+//    }
+//
+//    public List<Pokemon> getPokemons() {
+//        return pokemons;
+//    }
+//}
+
+import java.util.List;
+
+public class PokemonGymOwner {
```

Comment:
Deze klasse heeft `name` en `pokemons` overeenkomend met de `PokemonTrainer` klasse. Een PokemonGymOwner is in feite ook een PokemonTrainer, net zoals en sportschoolhouder ook zelf traint. Deze klasse mag dus overerven van PokemonTrainer.